project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=1.5.8
+sbt.version=1.6.2
```

Comment:
Was this intentional?

Reply:
[same](https://github.com/pagopa/interop-commons/pull/163#discussion_r913927175)